impl/src/test/java/org/ehcache/docs/GettingStarted.java

```diff
@@ -129,16 +132,24 @@ public void persistentCacheManager() {
   }
 
   @Test
-  public void offheapCacheManager() {
+  public void offheapCacheManager() throws InterruptedException {
     // tag::offheapCacheManager[]
     CacheManager cacheManager = CacheManagerBuilder.newCacheManagerBuilder().withCache("tieredCache",
         CacheConfigurationBuilder.newCacheConfigurationBuilder()
             .withResourcePools(ResourcePoolsBuilder.newResourcePoolsBuilder()
                 .heap(10, EntryUnit.ENTRIES)
                 .offheap(10, MemoryUnit.MB)) // <1>
+            .withExpiry(Expirations.timeToLiveExpiration(new Duration(10, TimeUnit.SECONDS)))
             .buildConfig(Long.class, String.class)).build(true);
 
-    cacheManager.close();
+    final Cache<Long, String> tieredCache = cacheManager.getCache("tieredCache", Long.class, String.class);
+    tieredCache.put(1L, "not expired yet...");
+    Thread.sleep(TimeUnit.SECONDS.toMillis(5));
```

Comment:
Yuck! Can this be moved to another class and not on what appears in asciidoc? Also it needs to properly use a time source and not `Thread.sleep()`